__tests__/app.test.js

```diff
@@ -353,4 +353,49 @@ describe("CORE: PATCH /api/articles/:article_id", () => {
                   expect(body).toEqual({Error : '400, Bad Request'})
               })
             })
-    })
\ No newline at end of file
+    })
+
+
+
+
+
+describe("CORE: DELETE /api/articles/:comment_id", () => {
+  test("200: Endpoint should contain deleted comment with correct id", () => {
+    return request(app)
+      .delete("/api/comments/2")
+      .expect(200)
+      .then(({body}) => {
+          expect(body.deleted_comment).toHaveProperty("comment_id", (2));
+          expect(body.deleted_comment).toHaveProperty("votes", expect.any(Number)); 
+          expect(body.deleted_comment).toHaveProperty("created_at", expect.any(String)); 
+          expect(body.deleted_comment).toHaveProperty("author", expect.any(String)); 
+          expect(body.deleted_comment).toHaveProperty("body", expect.any(String)); 
+          expect(body.deleted_comment).toHaveProperty("article_id", expect.any(Number)); 
+      });
```

Comment:
A delete should return a 204 status code - and a 204 cannot send over a body. **Just** expecting the error code is fine for a delete 'happy path' test - no need to check any body coming back.